### DISCUSSION TITLE
[Feature] Past x Upcoming flights should have different colors on map

### DISCUSSION THREAD

**Discussion by yurividal** [johanohly/AirTrail#42] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to see upcoming flights with a different color on the map. Easier to spot.

**Comment by johanohly** [comment]
That's a good idea. I haven't put much thought towards how to represent future flights, but this seems like a reasonable way to do it.

**Comment by johanohly** [comment]
Added in v0.2.0
